src/main/java/frc/robot/Constants.java

```diff
@@ -88,6 +88,12 @@ public static class ArmConstants {
    * Constants for the game piece manipulator
    */
   public static class GamePieceManipulatorConstants {
+    public static final AngularVelocity INTAKESPEED = RadiansPerSecond.of(5);
 
+    public static final AngularVelocity OUTTAKESPEED = RadiansPerSecond.of(-5);
+
+    public static final AngularVelocity SCORESPEED = RadiansPerSecond.of(5);
```

Comment:
We don't know what these numbers will be, but I think its safe to say scoring will need to be the same direction as ejecting/outtaking (negative)